### ISSUE TITLE
Resource "aws_default_network_acl" orphaned subnet_ids

### ISSUE THREAD

**Issue by jeremyciak** [terraform-aws-modules/terraform-aws-vpc#529]
I'm not sure what changed but it seems like something recently corresponds with the blurb here on [Managing Subnets in the Default Network ACL](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/default_network_acl#managing-subnets-in-the-default-network-acl) for the `aws_default_network_acl` resource. I am getting continuously prompted that the currently associated subnets will be removed. I believe this module either needs to populate the `subnet_ids` value for the `aws_default_network_acl` resource or the following `lifecycle` configuration on the resource is needed:

```
resource "aws_default_network_acl" "default" {
  # ... other configuration ...

  lifecycle {
    ignore_changes = [subnet_ids]
  }
}
```

**Comment by antonbabenko** [comment]
Fixed in #530 

[v2.60.0](https://github.com/terraform-aws-modules/terraform-aws-vpc/releases/v2.60.0) has been just released.

**Comment by lorengordon** [comment]
Hmm, so the linked PR ended up only working if all the subnets are created by this module. If someone creates a subnet outside the module, it gets associated with the default NACL, and you continue to get a persistent diff. I think for this particular resource, the lifecycle ignore_changes is actually a better option.

```
  lifecycle {
    ignore_changes = [subnet_ids]
  }
```
